src/scenes/level.js

```diff
@@ -0,0 +1,467 @@
+      .setOrigin(0, 0)
+      .setDepth(2);
+    this.inventory.button.setInteractive();
+    this.inventory.button.setAlpha(0.1);
+    this.inventory.button.on("pointerup", () => {
+      this.hideAllMenus();
+      this.inventory.button.setAlpha(0.1);
+    });
+
+    // Audio settings
+    let audioWindowX = 550;
+    let audioWindowY = 205;
+    this.audio.audioPage = this.add
+      .image(audioWindowX, audioWindowY, "audio-settings")
+      .setOrigin(0, 0)
+      .setDepth(1);
+    this.audio.audioPageButton = this.add
+      .image(660, 465, "audio-settings-button")
+      .setOrigin(0, 0)
+      .setDepth(2);
+    this.audio.audioPageButton.setInteractive();
+    this.audio.audioPageButton.on("pointerup", () => {
+      this.showAudioSettings(true);
+    });
+
+    // Place sliders
+    let barXOffset = 53;
+    this.audio.sliders = [];
+    this.audio.sliders.push(
+      this.add
+        .image(audioWindowX + barXOffset, audioWindowY + 80, "audio-slider")
+        .setOrigin(0, 0)
+        .setDepth(2)
+    );
+    this.audio.sliders.push(
+      this.add
+        .image(audioWindowX + barXOffset, audioWindowY + 125, "audio-slider")
+        .setOrigin(0, 0)
+        .setDepth(2)
+    );
+    this.audio.sliders.push(
+      this.add
+        .image(audioWindowX + barXOffset, audioWindowY + 170, "audio-slider")
+        .setOrigin(0, 0)
+        .setDepth(2)
+    );
+
+    // Set 5 buttons for each of the 3 sliders
+    this.audio.audioButtons = [];
+    for (let volumeType = 0; volumeType < 3; volumeType++) {
+      let audioButtonRow = [];
+      for (let buttonNum = 0; buttonNum < 5; buttonNum++) {
+        let audioButton = this.add
+          .image(
+            audioWindowX + barXOffset + 10 + buttonNum * 22,
+            audioWindowY + 80 + volumeType * 45,
+            "audio-button"
+          )
+          .setOrigin(0, 0)
+          .setDepth(3);
+        audioButton.setInteractive();
+        audioButton.setAlpha(0.1);
+        audioButtonRow.push(audioButton);
+        audioButton.on("pointerup", () => {
+          audioScene.changeVolume(volumeType, buttonNum);
+          this.changeAudioButton(volumeType, buttonNum);
+        });
+      }
+      // Save 2d array of buttons (3 x 5)
+      this.audio.audioButtons.push(audioButtonRow);
+    }
+    // Hide audio page on startup
+    this.showAudioSettings(false);
+
+    // Class picture
+    this.add
+      .image(0, 250, this.characterData.characterClass)
+      .setOrigin(0, 0)
+      .setDepth(2);
+
+    // Gold
+    this.goldText = this.add
+      .text(20, 20, "Gold: " + this.characterData.gold, {
+        fill: "gold",
+        fontSize: "30px"
+      })
+      .setDepth(3);
+
+    // Show stats
+    let statColor = "white";
+    this.enemiesKilledText = this.add
+      .text(
+        20,
+        60,
+        "Enemies killed: " + this.characterData.totalEnemiesKilled,
+        { fill: statColor }
+      )
+      .setDepth(3);
```

Comment:
```
this.enemiesKilledText = this.add
      .text(
        20,
        60,
        "Enemies killed: " + this.characterData.totalEnemiesKilled,
        { fill: statColor }
      )
      .setDepth(3);
```

This is taking it a bit too far

Reply:
I'm letting Prettier do its thing. I believe if it can't fit all parameters on one line then it splits them up like this. I agree with what it is doing.